lib/airbrake.js

```diff
@@ -183,11 +186,17 @@ Airbrake.prototype.notify = function (err, cb) {
     }
   });
 
-  if (exit) {
+  var notice = this.notifyJSON(err);
+
+  this.filters.forEach(function (filter) {
+    notice = filter(notice);
```

Comment:
By the way, we should probably modify `exit` here instead of `notice`. With this approach we won't need to amend line 195.